react-frontend/src/views/DoctorSignIn.js

```diff
@@ -86,7 +88,7 @@ function DoctorSignIn() {
                         </div>
                       </Grid>
                       <div>
-                        <Button type="submit" className="signInButton" style={{backgroundColor: '#CF6A6A', color: 'white', fontWeight: 'bold', fontSize: 17, borderRadius: 10}} variant="contained" onClick={checkCredentials}>LOGIN</Button>
+                        <Button type="submit" className="signInButton" style={{backgroundColor: '#CF6A6A', color: 'white', fontWeight: 'bold', fontSize: 17, borderRadius: 10}} variant="contained" onClick={(e) => checkCredentials(e)}>LOGIN</Button>
```

Comment:
Avoid defining such long inline styles, add it using BEM convention/ or make a different class inside CSS or add it to signInButton